docs/recipes/Making-An-Api-Call.md

```diff
@@ -4,15 +4,278 @@
+    if (response.ok) {
+      const data = await response.json();
+      store.dispatch({ type: 'LOADED_API', data });
+    } else {
+      store.dispatch({ type: 'FAILED_API' });
+    }
+  } catch (e) {
+    store.dispatch({ type: 'FAILED_API' });
+  }
+};
+
+Books.holocron = {
+  // Runs on both Server and Browser
+  loadModuleData,
+  reducer,
+};
+```
+
+The modules reducer would handle those dispatched actions so the module would be able to retrieve the data from the Redux store.
+
+## Duplicate Calls Across modules
+
+When adding response data to the redux store it can be tempting to try to access this directly in other modules. This is **not** a recommended approach as you should aim to have modules as independent as possible.
+
+You could also choose to bubble up your data fetching to the root module and pass down the data as props, which is a common approach with React applications, however this will result in closer coupling between child and root modules.
+
+## Fetchye
+
+[Fetchye](https://github.com/americanexpress/fetchye) brings a new, simplified method of making cached API calls. It makes use of React hooks to provide a simple API to enable data fetching with a centralized cache. Combined with the `fetchye-one-app` helpers it has minimal configuration and does not tightly couple a root module configuration to child modules.
+
+Below is a breakdown of the APIs used to integrate with One App:
+
+* [`useFetchye`](https://github.com/americanexpress/fetchye#usefetchye) from `fetchye` - A react hook responsible for dispatching an asynchronous fetch request to a given URL.
+* [`OneFetchyeProvider`](https://github.com/americanexpress/fetchye#onefetchyeprovider) from `fetchye-one-app` - This is a react context provider which will ensure that any `useFetchye` calls will use the One App configuration. Think of this as the global config for your application. It is not required for `useFetchye` to work but it enables `useFetchye` to de-dupe requests and make use of a centralized cache.
+* [`OneCache`](https://github.com/americanexpress/fetchye#onecache) from `fetchye-one-app` - This is a configured cache for use with One App modules. This is the cache which `OneFetchyeProvider` will always use.
+* [`makeOneServerFetchye`](https://github.com/americanexpress/fetchye#makeoneserverfetchye) from `fetchye-one-app` - This helper creates a specialized fetch client for making requests on the One App server for server side rendering.
+
+
+### Using `useFetchye`
+
+Install `fetchye`:
+
+```bash
+npm i -S fetchye
+```
+
+Updating the first example to use `useFetchye` reduces the amount of boilerplate required for handling the request, loading and error states.
+
+```jsx
+import { useFetchye } from 'fetchye';
+
+const BooksModule = () => {
+  const { isLoading, data, error } = useFetchye('https://some-data-server.com/api/v1/books');
+  const books = data && data.body;
+
+  if (isLoading) {
```

Comment:
should we also advice using `data.body.status===200`

Reply:
we could add a check for `data.ok` but that is something i expect an engineer to handle, this is just a basic example of the fetchye api